### PR TITLE
Dev max connections

### PR DESCRIPTION
**Pull request by michaelhasty**
Added maximum and active connection to host. Added atMaxConnections state.

### CODE REVIEW

File: src/main/java/requestManagement/fleetManager/hosts/Host.java [hunk 1]
```diff
@@ -35,6 +38,13 @@ public int getPort() {
         return port;
     }
 
+    /** Returns a int value of max connections for the host */
+    public int getMaxConnections () { return maxConnections; }
+
+    public int getActiveConnections() {
```
Comment (Tom-J-Barrett) [inline src/main/java/requestManagement/fleetManager/hosts/Host.java]: Why is getActiveConnections method (line 44) not in 1 line like method on line 42 getMaxConnections

Comment (michaelhasty) [reply src/main/java/requestManagement/fleetManager/hosts/Host.java]: Fixed thanks.

File: src/main/java/requestManagement/fleetManager/hosts/Host.java [hunk 2]
```diff
@@ -35,6 +38,15 @@ public int getPort() {
         return port;
     }
 
+    /** Returns a int value of max connections for the host */
+    public int getMaxConnections() {
+        return maxConnections;
+    }
+
```
Comment (RobHoodless) [inline src/main/java/requestManagement/fleetManager/hosts/Host.java]: javadoc :)

File: src/main/java/requestManagement/fleetManager/hosts/Host.java [hunk 3]
```diff
@@ -54,9 +67,13 @@ public void setState(String state) {
         this.state = new HostStateFactory().hostState(state);
     }
 
+
     public boolean isActive() {
         return this.getState().toString().equals("active");
     }
```
Comment (Cribbin) [inline src/main/java/requestManagement/fleetManager/hosts/Host.java]: L73: Add space between `isActive()` and `setActiveConnections()`

File: src/main/java/requestManagement/fleetManager/hosts/Host.java [hunk 4]
```diff
@@ -54,9 +67,13 @@ public void setState(String state) {
         this.state = new HostStateFactory().hostState(state);
     }
 
+
```
Comment (Cribbin) [inline src/main/java/requestManagement/fleetManager/hosts/Host.java]: L70: Remove this unnecessary space

File: src/main/test/requestManagement/fleetManager/storage/HostDAOFileSystemTest.java [hunk 5]
```diff
@@ -29,8 +30,8 @@ void setUp() {
         System.out.println(file);
         dao = new HostDAOFileSystem(file);
 
-        Host host1 = getHost(VALID_PRIVATE_IP_1, DNS_NAME_1, ACTIVE_STATE);
-        Host host2 = getHost(VALID_PRIVATE_IP_2, DNS_NAME_2, INACTIVE_STATE);
+        Host host1 = getHost(VALID_PRIVATE_IP_1, DNS_NAME_1, ACTIVE_STATE, 10);
```
Comment (Cribbin) [inline src/main/test/requestManagement/fleetManager/storage/HostDAOFileSystemTest.java]: L33-34, L52-53: These are what are known as [magic numbers](https://en.wikipedia.org/wiki/Magic_number_(programming)). Can you make a `private static final int` for these at the top like the rest of the variables so it's clearer to a reader what the 10 and 12 represent? It'll make for easier to read tests.

File: src/main/java/requestManagement/fleetManager/hosts/Host.java [hunk 6]
```diff
@@ -82,7 +82,7 @@ public String toString() {
             .append(",").append(this.getDns())
             .append(",").append(this.getPort())
             .append(",").append(this.getState())
-            .append(",").append(this.getSubnet())
+            .append(",").append(this.getMaxConnections())
```
Comment (RobHoodless) [inline src/main/java/requestManagement/fleetManager/hosts/Host.java]: Are we no longer storing subnet?

**Pull request merged**